languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -12209,6 +12209,99 @@ USA
+                <example>Área marinha protegida é um termo que se refere a um largo conjunto de regimes de protecção territorial a nível de zonas costeiras e insulares, com...</example>
+           </antipattern>
+
+           <antipattern>
+                <token postag='(SPS00:)?D[AI].+' postag_regexp='yes'/>
+                <token postag='NC.+' postag_regexp='yes'/>
+                <token>que</token>
+                <token>se</token>
+                <token postag='V.+' postag_regexp='yes'/>
+                <token postag='V.+' postag_regexp='yes'/>
+                <token min='0' max='1' postag='SPS00' postag_regexp='no'/>
+                <token postag='VMN0000' postag_regexp='no'/>
+                <example>O cladograma que se segue permite visualizar e comparar as relações evolucionárias entre os diversos táxons relacionados, dev...</example>
+                <example>os riachos que se subdividem continuam a fluir no escuro" e "A fala comum e refinada juntam-se no escuro, frases claras e obscur...</example>
+           </antipattern>
+
+           <pattern>
+                <token postag='(SPS00:)?D[AI].+' postag_regexp='yes'>
+                    <exception regexp='yes'>d[ao]s?|dum|n[ao]s?|pel[ao]s?</exception>
+                    <exception postag_regexp='yes' postag='SPS00:PI.+'/> <!-- "nessa" -->
+                    <exception scope='previous' postag_regexp='yes' postag='VMN0000|CS|SPS00'/>
+                    <exception scope='previous' regexp='yes'>e|toda</exception>
+                </token>
+                <token postag='NC[MF].+' postag_regexp='yes'>
+                    <exception postag_regexp='yes' postag='RG|SPS0.+'/>
+                </token>
+                <marker>
+                    <token>que</token>
+                    <token>se</token>
+                    <token postag='VMIP3.+' postag_regexp='yes'/>
+                </marker>
+                <token postag='V.+|SPS0.+|DA.+|NC.+|AQ.+' postag_regexp='yes'>
+                    <exception postag_regexp='yes' postag='VMI[FS].+|VMG0000|VMP00.+|VMN0000|DI.+|CS|SPS00:PP.+'/>
+                    <exception regexp='yes'>com|d[ao]s?|de|pel[ao]s?|sobre</exception>
+                </token>
+                <token>
+                    <exception postag_regexp='yes' postag='DA.+|Z0.+'/>
+                    <exception scope='next' postag_regexp='yes' postag='DA.+|CS'/>
+                </token>
+            </pattern>
+            <filter class="org.languagetool.rules.pt.AdvancedSynthesizerFilter" args="lemmaFrom:5 lemmaSelect:V.* postagFrom:2 postagSelect:NC(.)(.).* postagReplace:VMP00[\b2][\b1].*" />
+            <message>&simplify_msg;</message>
+            <suggestion>{suggestion}</suggestion>
+            <example correction="levado">O Rui calcula o tempo <marker>que se leva</marker> a percorrer cada cidade.</example>
```

Comment:
This is ungrammatical in `pt-BR`. 'O tempo levado' sounds like 'naughty time'.

Reply:
@p-goulart 

Maybe make it pt-PT only?

Reply:
If we can't find any examples of this being grammatical in `pt-BR` but we can in `pt-PT`, then I suppose we could make it dialect-specific, yes.